docker-compose-setup/docker-compose.yml

```
@@ -0,0 +1,54 @@
+services:
+  coatrack-postgres:
+    image: postgres:9.4.26
+    container_name: coatrack-postgres
+    volumes:
+      - coatrack-postgres:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: password1234
+  coatrack-config-server:
+    image: coatrack/coatrack-config-server:2.0.0-SNAPSHOT
+    container_name: coatrack-config-server
+    depends_on:
+      - coatrack-postgres
+    ports:
+      - 8998:8998
+    entrypoint: ["sh", "-c", "java -Djava.security.egd=file:/dev/./urandom \
+      -jar /opt/coatrack/lib/coatrack-config-server.jar \
+      --spring.datasource.url=jdbc:postgresql://coatrack-postgres/ygg_config_server \
+      --spring.datasource.password=password1234"]
+  coatrack-admin:
+    image: coatrack/coatrack-admin:2.0.0-SNAPSHOT
+    container_name: coatrack-admin
+    volumes:
+      - coatrack-admin:/home
+    depends_on:
+      - coatrack-config-server
+      - coatrack-postgres
+    ports:
+      - 8080:8080
+    entrypoint: [ "sh", "-c", "sleep 5 && java -Djava.security.egd=file:/dev/./urandom \
+      -jar /opt/coatrack/lib/coatrack-admin.jar \
+      --ygg.admin.gitService.url=https://bitbucket.org/perezdf/anthill-settings.git \
+      --ygg.admin.gitService.user=fpd.judo@gmail.com \
+      --ygg.admin.gitService.password=fpdvip1234 \
+      --spring.datasource.url=jdbc:postgresql://coatrack-postgres/coatrack \
+      --spring.datasource.username=postgres \
+      --spring.datasource.password=password1234 \
+      --spring.jpa.hibernate.ddl-auto=update \
+      --spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect \
+      --spring.datasource.initialization-mode=always \
+      --ygg.admin.database.insertSampleDataOnStartup=false \
+      --ygg.proxy.generate-bootstrap-properties.spring.cloud.config.uri=http://localhost:8998 \
+      --spring.cloud.config.uri=http://coatrack-config-server:8998 \
+      --spring.cloud.config.username=admin \
+      --spring.cloud.config.password=password \
+      --logging.level.eu.coatrack.proxy=DEBUG \
```

This package is not part of coatrack-admin, it should probably be: 
```suggestion
      --logging.level.eu.coatrack.admin=DEBUG \
```

---

I just removed this parameter. This was only used during development of this feature.